src/normalizations/normalize-conversation.ts

```diff
@@ -55,7 +55,7 @@ export const normalizeConversation = ({
 			date: c.d,
 			messages,
 			participants: c.e ? map(c.e, normalizeParticipantsFromSoap) : undefined,
-			subject: !isNil(c.su) ? c.su : '',
+			subject: c.su,
```

Comment:
I agree on the fix as a temporary solution because the original fix implementation faced some corner case which is difficult to handle and requires more effort.
IMO the existing problem in the codebase is that the type doesn't reflect the actual data: the message returned by the BE can not contain the _subject_ field (also documented in the API docs), while the _MailMessage_ type defines the _subject_ as an NON optional string property.
The proper solution should be to fix the _MailMessage_ definition and subsequently change all the code that use the _subject_ to handle the case of the missing property.
I already created the TI for that (IRIS-4907)